src/table/utils/styling-utils.js

```diff
@@ -153,6 +153,28 @@ export function getBodyCellStyle(layout, theme) {
   };
 }
 
+/**
+ * Get total head cell style
+ *
+ * @param {Object} totalHeaderStyle
+ * @param {Object} layout
+ * @returns total cell style regarding the total head position
+ */
+export function getTotalsCellStyle(totalHeaderStyle, layout) {
+  if (getTotalPosition(layout) === 'noTotals') return false;
+  totalHeaderStyle.borderColor = 'rgb(166, 166, 166)';
```

Comment:
I don't think you want to change this, since the border color depends on the theme

Reply:
no not if you do it at this stage. What comes from `getHeaderStyle` is the final style, theme taken into account